docs/docs/05-reusable-components.md

```diff
@@ -227,7 +227,31 @@ Counter.defaultProps = { initialCount: 0 };
 
 ### No Autobinding
 
-Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or arrow functions.
+Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or [arrow](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) `=>` functions.
+
+The example below illustrates `.bind(this)` being used to bind a method in the constructor.
+
+```javascript
+class Newsfeed extends React.Component {
+  constructor() {
+    super();
+    this.changeContent = this.changeContent.bind(this);
```

Comment:
The perf hit can be pretty bad, especially at scale. We actually even mentioned this exact pattern for people who really want autobinding.

Now that said, I don't think we need this example here right now. Let's keep this PR focused on grammar updates.